src/bosh_azure_cpi/spec/unit/azure_client/list_network_interfaces_by_keyword_spec.rb

```diff
@@ -26,6 +26,28 @@
   let(:valid_access_token) { 'valid-access-token' }
   let(:expires_on) { (Time.new + 1800).to_i.to_s }
 
+  describe '#get_resources_by_url' do
```

Comment:
Why we have the describe `#get_resources_by_url`  with tests which are empty?

Reply:
This code is related to #652 (areas of un-covered code which are already in master), and is technically unrelated to either #644 or this PR (other than the fact that they bring coverage down for this and all other PRs until #652 is fixed and coverage returns to 100%, which is stated as a requirement in the template for all PRs).

I had started an attempt to return coverage to 100% within this PR, but found that doing so would take too much time. So I left 9 Pending/unimplemented specs to be addressed later separately (see #652 for details).

Reply:
I will suggest to remove the empty test cases because we want to change the requirement to have 100% code coverage.

Reply:
removed as requested